crates/ef-testing/src/models/case.rs

```diff
@@ -184,7 +188,7 @@ impl BlockchainTestCase {
                     continue;
                 }
                 Some(state) => {
-                    assert_contract_post_state(test_case_name, evm_address, expected_state, state)?
+                    assert_contract_post_state(test_case_name, evm_address, expected_state, state)?;
```

Comment:
here clippy says we don't return anything in this arm, so it's bad practice to not put a semicolon, since we're returning into the void